include/at_chops/aes_ctr.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#ifdef BUILD_MBEDTLS
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+#include <string.h>
+
+typedef struct {
+    unsigned char *res; // result of the encryption/decryption
+    size_t reslen; // length of the result written
```

Comment:
Usually the size / len variable comes first in the struct when you have variable length pointers in a struct, that way you can compute where things are located in the memory.

Reply:
Done in [036fa72](https://github.com/atsign-foundation/at_c/pull/7/commits/036fa7202f72c3989ce7014d94ef70b3c8398f80)

Reply:
Interesting, I did not know this. !